components/ShoppingList.js

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+
+import ListItem from "./ListItem";
+
+export default function ShoppingList({ items }) {
+  return (
+    <div>
+      <ul>
+        {items
+          .filter((item) => !item.checked)
+          .map(({ id, name, checked }) => (
+            <ListItem key={id} name={name} isChecked={checked} />
+          ))}
+      </ul>
+      <ol>
+        {items
+          .filter((item) => item.checked)
+          .map(({ id, name, checked }) => (
+            <ListItem key={id} name={name} isChecked={checked} />
+          ))}
+      </ol>
+    </div>
```

Comment:
Why do you use first an ul and afterwards an ol?

Reply:
Because an unchecked item from the first list shall switch to the second list, when checked. Thereby, this item shall be displayed at the top of the second list. I anticipated, that I would need an ol for that to be accomplished?

Reply:
No you don't need an ol for that, the only difference between ol and ul is that ol has numbers, ul has only dashes. So please change it to ul.